### PR TITLE
Separate composefiles

### PR DESCRIPTION
**Pull request by Pninja**
1.  Adds separate composefile and instructions for windows users (using external volume) and goes back to normal volumes on main docker-compose.yml file. Those users that have already configured the external volume, can still use it by running: 
`docker-compose -f docker-compose-win.yml up`
when starting the project.

2. Updates node packages

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,26 +1,30 @@
...
+            set -x
+            mkdir -p ./volumes/mattermost/{data,logs,config,plugins}
+            sudo chown -R 2000:2000 ./volumes/mattermost/
+            docker-compose up -d
+      - run:
+          name: Check if db is ready
+          command: |
+            docker ps -a
+            docker logs project_db_1
+            docker exec -i project_db_1 pg_isready -U $POSTGRES_USER -d $POSTGRES_DB
```
Comment (larenala) [inline .circleci/config.yml:19]: At this point CircleCi tests fail. Based on error message it's probably something to do with missing settings for AWS, or maybe the env-parameters should be added to CircleCi?

File: README-EN.md [hunk 2]
```diff
@@ -42,30 +41,28 @@ After the containers are up you need to create the database itself and run dumps
 
 'docker exec -i kohdataan-backend_db_1 createdb -U mmuser database_test'
 
-
 After this go to db/dumps and run commands;
 
 'docker exec -i kohdataan-backend_db_1 psql -U mmuser kohdataan < kohdataan.pgsql'
 
 'docker exec -i kohdataan-backend_db_1 psql -U mmuser mattermost < mattermost.pgsql'
```
Comment (larenala) [inline README-EN.md]: Just a tiny nit: in the Finnish Readme these are formatted as code, which for me at least feels easier to read. But maybe just personal preference :)

File: README-EN.md [hunk 3]
```diff
@@ -119,12 +116,12 @@ The project adheres to Web Content Accessibility Guidelines (WCAG) 2.1 accessibi
...
-  * etc.
-* 
+- Navigating to another view
+- Messages:
+  - New message is sent
+  - New message is received
+  - etc.
+-
 
 Automated accessibility testing tools are not perfect. Before submitting a pull request, please check that your work is not in conflict with the following accessibility checklist: <checklist TBA>
```
Comment (larenala) [inline README-EN.md:136]: Link is missing here. Also this section doesn't exist in the Finnish Readme for some reason.

**Review by larenala — COMMENTED** [review]
Seems to work fine after the initial set-up trouble! I added just a few comments. Unfortunately I don't have Windows so I can't test the instructions in practice.

File: .circleci/config.yml (hunk 1 above)
Comment (Pninja) [reply .circleci/config.yml:19]: Yes, tests are failing also in master and we have not been able to figure out how to fix it. :((

**Pull request merged**